kitsune/sumo/static/sumo/less/main.less

```diff
@@ -307,6 +307,27 @@ input[type="submit"]{
   }
 }
 
+/* Taken from Bootstrap V3 */
+.alert {
+  border: 1px solid transparent;
+  border-radius: 4px;
+  background: #FFE url("../img/warning-stripes.png") repeat-x scroll left top;
```

Comment:
To clarify: Putting CSS rules in alphabetical order is good style, and actually has a small impact on the compressed file size. Smaller files means faster pages. So it seems silly, but it helps.